.teamcity/settings.kts

```diff
@@ -305,6 +306,23 @@ object RunAllUnitTests : BuildType({
 				}
 			}
 		}
+
+		notifications {
+			notifierSettings = slackNotifier {
+				connection = "PROJECT_EXT_11"
```

Comment:
These `PROJECT_EXT_*` things seems to be project config inherited from the `<Root>` project in TC. I don't know how to verify this value is correct, other than trusting that the UI generated the right DSL.